pages/Iron-Dust.md

```diff
@@ -0,0 +1,19 @@
+Iron Dust is one of the nine [dusts](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Dusts) added in Slimefun.
+
+## Obtaining
+Crushing Iron Ore in an [Ore Crusher](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ore-Crusher) or an [Electric Ore Grinder](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ore-Grinder) yields two pieces of Iron Dust, effectively doubling it.
+
+Iron Dust has a ~11.11% chance of being extracted from [Sifted Ore](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Sifted-Ore), by processing it through an [Ore Washer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ore-Washer) or an [Electric Dust Washer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Dust-Washer).
+
```

Comment:
You could maybe also mention that you can obtain it through "Iron Ore chunks" too, if you are using SlimefunOreChunks